docs/src/modules/java/pages/setup-and-dependency-injection.adoc

```diff
@@ -23,6 +23,19 @@ It is important to remember that an Akka service consists of one to many distrib
 individually and independently, for example during a rolling upgrade. Each such instance starting up will invoke
 `onStartup` when starting up, even if other instances run it before.
 
+== Disabling components
+
+Users can use `ServiceSetup` to disable components by overriding `disabledComponents` and returning a set of component classes to disable.
```

Comment:
"Users" is written from our (Akka's) perspective, but we should write from the reader's perspective. We are targeting developers so I think this would be better as "You can use"

Reply:
fixed